.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "0.12"
+  - "5.9.1"
```

Comment:
I agree that 4.x should fine moreover for Travis ! You can use 5.x on your computer.